Makefile

```diff
@@ -310,6 +312,44 @@ e2e-remove-resources:
 
 .PHONY: test-mutation
 
+###############################################################################
+###                                Docker                                  ###
+###############################################################################
+
+RUNNER_BASE_IMAGE_DISTROLESS := gcr.io/distroless/static
+RUNNER_BASE_IMAGE_ALPINE := alpine:3.16
+RUNNER_BASE_IMAGE_NONROOT := gcr.io/distroless/static:nonroot
+
+docker-build:
+	@DOCKER_BUILDKIT=1 docker build \
+		-t osmosis:local \
+		-t osmosis:local-distroless \
+		--build-arg GO_VERSION=$(GO_VERSION) \
+		--build-arg RUNNER_IMAGE=$(RUNNER_BASE_IMAGE_DISTROLESS) \
+		--build-arg GIT_VERSION=$(VERSION) \
+		--build-arg GIT_COMMIT=$(COMMIT) \
+		-f Dockerfile .
+
+docker-build-distroless: docker-build
+
+docker-build-alpine:
+	@DOCKER_BUILDKIT=1 docker build \
+		-t osmosis:local-alpine \
+		--build-arg GO_VERSION=$(GO_VERSION) \
+		--build-arg RUNNER_IMAGE=$(RUNNER_BASE_IMAGE_ALPINE) \
+		--build-arg GIT_VERSION=$(VERSION) \
+		--build-arg GIT_COMMIT=$(COMMIT) \
+		-f Dockerfile .
+
+docker-build-nonroot:
```

Comment:
Does `docker-build-debug` need to be updated?

Reply:
Yes, but not because of these changes.
There is no longer a "debug" osmosis image. 

I have opened this issue to remove it: https://github.com/osmosis-labs/osmosis/issues/2853